utils/SettingsUtils.tsx

```diff
@@ -49,6 +53,122 @@ export function setSetting(key: any, value: any) {
     localStorage.setItem(LOCAL_STORAGE_SETTINGS_KEY, JSON.stringify(settings))
 }
 
+export function setSettingsChangedData(data: any): Promise<void> {
+    return new Promise(resolve => {
+        data.visibility = data.visibility || {}
+        data.mod = data.mod || {}
+        data.filters = data.filters || {}
+
+        setSetting(
+            FLIP_CUSTOMIZING_KEY,
+            JSON.stringify({
+                hideCost: !data.visibility.cost,
+                hideEstimatedProfit: !data.visibility.estProfit,
+                hideLowestBin: !data.visibility.lbin,
+                hideSecondLowestBin: !data.visibility.slbin,
+                hideMedianPrice: !data.visibility.medPrice,
+                hideSeller: !data.visibility.seller,
+                hideVolume: !data.visibility.volume,
+                maxExtraInfoFields: data.visibility.extraFields,
+                hideProfitPercent: !data.visibility.profitPercent,
+                hideSellerOpenBtn: !data.visibility.sellerOpenBtn,
+                hideLore: !data.visibility.lore,
+                useLowestBinForProfit: data.lbin,
+                shortNumbers: data.mod.shortNumbers,
+                soundOnFlip: data.mod.soundOnFlip,
+                justProfit: data.mod.justProfit,
+                blockTenSecMsg: data.mod.blockTenSecMsg,
+                hideModChat: !data.mod.chat,
+                modFormat: data.mod.format,
+                modCountdown: data.mod.countdown,
+                disableLinks: !data.visibility.links,
+                hideCopySuccessMessage: !data.visibility.copySuccessMessage,
+                finders: FLIP_FINDERS.filter(finder => {
+                    return hasFlag(parseInt(data.finders), parseInt(finder.value))
+                }).map(finder => parseInt(finder.value))
+            } as FlipCustomizeSettings)
+        )
+
+        setSetting(
+            FLIPPER_FILTER_KEY,
+            JSON.stringify({
+                maxCost: data.maxCost,
+                minProfit: data.minProfit,
+                minProfitPercent: data.minProfitPercent,
+                minVolume: data.minVolume,
+                onlyBin: data.filters.Bin === 'true'
```

Comment:
as you requested this got changed to 
```suggestion
                onlyBin: data.onlyBin
```